audit_exceptions/mismatched_binary_allowlist.json

```diff
@@ -1,4 +1,5 @@
 [
+  "balena-cli",
```

Comment:
Why is this still needed?

Reply:
This relates to last comment for [`node-raspberrypi-usbboot`](https://github.com/balena-io-modules/node-raspberrypi-usbboot) ELF files, which causes audit errors on Linux.

Reply:
It would be nicer to have a granular allowlist that ignores particular directories or files, e.g.
`"balena-cli": ["libexec/lib/node_modules/balena-cli/node_modules/node-raspberrypi-usbboot"]`

In mean time, this is needed on Linux as `balena-cli` interacts with other devices like RPi or Intel NUC (https://www.balena.io/docs/reference/balena-cli/#devices-supported)

I think the RPi USBboot feature is https://www.balena.io/docs/reference/balena-cli/#local-flash-image
> Flash a balenaOS image to a drive. Image file may be one of: .img|.zip|.gz|.bz2|.xz

Reply:
> It would be nicer to have a granular allowlist that ignores particular directories or files

Yes, something like this would be useful.